EndlessClient/Dialogs/TextMultiInputDialog.cs

```diff
@@ -154,9 +201,68 @@ protected override void OnDrawControl(GameTime gameTime)
             _spriteBatch.Begin();
             _spriteBatch.Draw(_background, DrawAreaWithParentOffset, _backgroundSourceRectangle, Color.White);
             _spriteBatch.Draw(_background, DrawPositionWithParentOffset + _bottomOverlayDrawPosition, _bottomOverlaySource, Color.White);
+            _spriteBatch.Draw(_background, DrawPositionWithParentOffset + _rightOverlayDrawPosition, _rightOverlaySource, Color.White);
+
+            _spriteBatch.Draw(_background, DrawPositionWithParentOffset + _rightOverlayDrawPosition, _rightOverlaySource, Color.White);
+
             _spriteBatch.End();
 
             base.OnDrawControl(gameTime);
         }
+
+        // OnUpdateControl needs to solely run if Dialog.Nine is in use
+
+        protected override void OnUpdateControl(GameTime gameTime)
+        {
+            if (_dialogSize != DialogSize.NineWithScroll)
+            {
+                base.OnUpdateControl(gameTime);
+                return;
+            }
+            _scrollBar.UpdateDimensions(9);
```

Comment:
This should be part of the initialization logic for the scrollbar in the constructor, rather than on every update. The number of items being scrolled doesn't change so `UpdateDimensions` doesn't need to be called more than once